src/commands/bot/giftcode/generate.js

```diff
@@ -0,0 +1,39 @@
+const { CommandStructures, SwitchbladeEmbed, Constants } = require('../../../')
+const { Command, CommandParameters, NumberParameter, CommandRequirements } = CommandStructures
+
+module.exports = class GiftCodeGenerate extends Command {
+  constructor (client, parentCommand) {
+    super(client, parentCommand || 'giftcode')
+    this.name = 'generate'
+    this.aliases = ['gnr']
+
+    this.parameters = new CommandParameters(this,
+      new NumberParameter(({ min: 25000, max: 50000, missingError: 'commands:giftcode.subcommands.generate.noValue' }))
+    )
+
+    this.requirements = new CommandRequirements(this, { guildOnly: true, databaseOnly: true, openDms: true, onlyOldAccounts: true })
+  }
+
+  async run ({ t, author, channel }, value) {
+    const embed = new SwitchbladeEmbed(author)
+    const Uembed = new SwitchbladeEmbed(author)
```

Comment:
that is unnecesary

Reply:
thats necessary

Reply:
like, its 2 embeds, one sends to author and the another one sends to the chat

Reply:
it is unnecessary. you don't need to declare it, just put new SwitchbladeEmbed inside .send

Reply:
the uembed is actually more easily, and i just copied from the last pr